src/coreclr/nativeaot/Runtime/EHHelpers.cpp

```diff
@@ -184,6 +184,7 @@ EXTERN_C void REDHAWK_CALLCONV RhpFailFastForPInvokeExceptionCoop(intptr_t PInvo
                                                                   void* pExceptionRecord, void* pContextRecord);
 int32_t __stdcall RhpVectoredExceptionHandler(PEXCEPTION_POINTERS pExPtrs);
 
+// REVIEW: this is no longer used by pInvokes and use in hijack seems bogus. Remove?
```

Comment:
Makes sense, I will log an issue. 
But we do not need this for hijack probes, right? There is no scenario that I can think of where we expect exceptions from waiting for GC.

Reply:
It may be there so that fatal crashes like stackoverflow or access violations caused by process state corruptions terminate the process immediately instead letting it to propagate.

Reply:
logged: https://github.com/dotnet/runtime/issues/73429